.mailmap

```diff
@@ -0,0 +1,13 @@
+Edwin Young <edwin@bathysphere.org>
+Edwin Young <edwin@bathysphere.org> catenary <catenary@users.sourceforge.net>
+Edwin Young <edwin@bathysphere.org> edwin young <catenary@edwins-MacBook-Air.local>
+Edwin Young <edwin@bathysphere.org> Tim Whidbey <catenary@bathysphere.org>
+Cameron Garnham <me@da2ce7.com>
+Richard Mant <dx-mon@users.sourceforge.net> Richard Mant <dx-mon@DX-TOP.(none)>
```

Comment:
Could I get you to change this please as it dead-names me? Thanks

Reply:
@DX-MON You would like to be called `dx-mon <dx-mon@users.sourceforge.net>` without the pseudonym 'Richard Mant'?

Reply:
Ahh, I get it; now your called: 'Rachel Mant', my apologies. I will make a corrective pull request.

Reply:
Correct, thank you :)

Reply:
But as always, as you like. Maybe you would prefer just `dx-mon`.

Reply:
@DX-MON https://github.com/fract4d/gnofract4d/pull/75 :)